src/StackExchange.Redis/Interfaces/IDatabase.cs

```diff
@@ -1272,7 +1272,8 @@ public interface IDatabase : IRedis, IDatabaseAsync
         /// <param name="values">The values to execute against.</param>
         /// <param name="flags">The flags to use for this operation.</param>
         /// <returns>A dynamic representation of the script's result.</returns>
-        /// <remarks><seealso href="https://redis.io/commands/evalsha"/></remarks>
+        /// <remarks>Please do not use this overload as it's not resilient to Redis server restarts. Use <see cref="ScriptEvaluate(string, RedisKey[], RedisValue[], CommandFlags)"/> instead.</remarks>
```

Comment:
I think it is fine to offer this as an *advisory comment*, but: I'd hesitate to say "please do not" - more just something that gives relevant warning; ditto on the EditorBrowsableState.Never

Reply:
Done.